package.json

```diff
@@ -165,7 +165,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run prettier:check"
+      "pre-commit": "npm run prettier:check && npm run lint"
```

Comment:
@gavinbarron , added a lint check but I wondered if I should also add a build check to prevent failed build checks

Reply:
That would be heavy on commits. Most builds take a couple of minutes I would advise against adding it there.